assets/src/components/Label.js

```diff
@@ -14,20 +32,27 @@ function Label (props) {
         position: 'absolute',
         display: 'inline-block',
         color,
-        left: labelPlacement === 'down'
-          ? `calc(${left} - 115px)`
-          : left,
-        zIndex: 1,
-        top: labelPlacement === 'down'
-          ? '55px'
-          : labelPlacement === 'downLower'
-            ? '75px'
-            : labelPlacement === 'up'
-              ? '-25px'
+        width: `${getWidth(value)}`,
+        textAlign: `${getTextAlign(value)}`,
+        left: `${getLeft(value)}`,
+        right: `${getRight(value)}`,
+        zIndex: labelPlacement === 'up1'
+          ? '3'
+          : labelPlacement === 'down1'
+            ? '2'
+            : labelPlacement === 'down2'
+              ? '1'
+              : '0',
+        top: labelPlacement === 'up1'
+          ? '-25px'
+          : labelPlacement === 'down1'
+            ? '55px'
+            : labelPlacement === 'down2'
+              ? '75px'
               : 0
       }}
     >
-      {labelText}
+      &nbsp;{labelText}&nbsp;
```

Comment:
Codacy is complaining about this saying unnecessary block? I tried to find a fix my self, but I couldn't figure out.I tried all {` `}